src/components/University/Courses/Styles.ts

```diff
@@ -0,0 +1,56 @@
+import {Link as ULink} from 'react-router-dom';
+import styled from 'styled-components';
+
+import {colors} from 'styles';
+import UUserLabel from 'components/UserLabel';
+
+export const Container = styled.div`
```

Comment:
tbh, I don't like the css code in here, since I'm not very good at CSS, maybe I have relied too much on third party libraries (bootstrap, materialUI etc) 🤣
so please feel free to point out CSS issues.